### ISSUE TITLE
Video not playing

### ISSUE THREAD

**Issue by laxmanrizecorp** [tcking/GiraffePlayer#27]
I downloaded source code of GiraffePlayer in github.

https://github.com/tcking/GiraffePlayer

But it is not working on default youtube or any other url's we given. We gave RTMP and HLS video' as well. It's return's IO exception.

Already we used in the existing url's in the sample app as well. Those videos also not working.

If possible, Can you please provide the working link of GiraffePlayer.

Thanks,
Lakshman kumar.M

**Comment by tcking** [comment]
is the device connect to network? or have you set any proxy ?

**Comment by laxmanrizecorp** [comment]
Yes. device connected to Wifi network. Same video's play in the VItamio player and normal android default(Media) player. but this Player is not working any more.

can you please check the below url's once.

http://76.12.161.171/nh1/04-12-2016_10_29_35.mp4/playlist.m3u8
http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4

We got the below error's for playing the videos
 http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4: I/O error
04-13 14:15:15.499: E/tv.danmaku.ijk.media.player.IjkMediaPlayer(6326): Error (-10000,0)

(14:15) maheswari: 04-13 14:15:38.334: E/IJKMEDIA(6326): http://76.12.161.171/nh1/04-12-2016_10_29_35.mp4/playlist.m3u8: Permission denied
04-13 14:15:38.344: E/tv.danmaku.ijk.media.player.IjkMediaPlayer(6326): Error (-10000,0)

Thanks,
Lakshman kumar.M

**Comment by diaconuliviu85** [comment]
<uses-permission android:name="android.permission.INTERNET" />
    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
 add this from manifest , and from android 6 
http://developer.android.com/intl/es/training/permissions/requesting.html

**Comment by laxmanrizecorp** [comment]
Hi,

I am getting issue like this
04-13 16:58:04.275: A/libc(3506): @@@ ABORTING: LIBC: ARGUMENT IS INVALID HEAP ADDRESS IN dlfree addr=0x4daa4e68
04-13 16:58:04.275: A/libc(3506): Fatal signal 11 (SIGSEGV) at 0xdeadbaad (code=1), thread 3546 (ff_read)

Thanks,
Lakshman kumar.M

**Comment by laxmanrizecorp** [comment]
and one more issue is like this

04-13 16:45:35.871: A/libc(13499): invalid address or address of corrupt block 0xb72a2290 passed to dlfree
04-13 16:45:35.871: A/libc(13499): Fatal signal 11 (SIGSEGV), code 1, fault addr 0xdeadbaad in tid 13577 (Thread-3894)

Thanks,
Lakshman kumar.M

**Comment by diaconuliviu85** [comment]
Tested on x86 cpu? if yes is a problem trying to arm. and follow up with details.

**Comment by APZ-87** [comment]
I can't play video from firebase storage 
via tcking giraffe player2 although it can download fully 100% , the error message is "something error" .
